index.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Programmers Survey Form</title>
+    <link rel="stylesheet" type="text/css" href="css/style.css">
+</head>
+<body>
+    <div id="heading">
```

Comment:
Using HTML file semantic tags over div's is recommended because the search engine is going to determine the importance of a web page using these semantic tags. Replacing <div> with <header> is a good practice to consider.